src/main/java/rtg/world/gen/ChunkProviderRTG.java

```diff
@@ -311,7 +284,8 @@ public Chunk provideChunk(final int cx, final int cz) {
                 baseBiomesList[k] = biomePatcher.getPatchedBaseBiome("" + Biome.getIdForBiome(landscape.biome[k].baseBiome));
             }
         }
-
+        volcanoGenerator.generateMapGen(primer, worldSeed, worldObj, cmr, mapRand, cx, cz, rtgWorld.simplex, rtgWorld.cell, landscape.noise);
```

Comment:
This is the point where we should be checking if volcanos are enabled in the configs.
If enabled, run the generator, if not, kick rocks.